vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/boutiquesParsing/BoutiquesParser.java

```diff
@@ -66,6 +66,16 @@ public BoutiquesApplication parseApplication(String descriptor) throws InvalidBo
         application.setSchemaVersion(getStringValue(parsedDescriptor, "schema-version"));
         application.setChallengerEmail(getStringValue(parsedDescriptor, "vip:miccai-challenger-email", true));
         application.setChallengerTeam(getStringValue(parsedDescriptor, "vip:miccai-challenge-team-name", true));
+        // Check if "VIPdot" values are one of the input IDs
+        JSONObject customObject = getObjectValue(parsedDescriptor, "custom", true);
+        if (customObject != null && customObject.containsKey("VIPdot")) {
```

Comment:
There is already a `getArrayValueAsStringSet` to do that

Reply:
used getArrayValueAsStringSet instead of getstringValue